oci/oci.go

```diff
@@ -179,14 +184,51 @@ func (e ExecSyncError) Error() string {
 
 // ExecSync execs a command in a container and returns it's stdout, stderr and return code.
 func (r *Runtime) ExecSync(c *Container, command []string, timeout int64) (resp *ExecSyncResponse, err error) {
-	args := []string{"exec", c.name}
+	parentPipe, childPipe, err := os.Pipe()
```

Comment:
I'd create a function in the `oci` package to wrap all of this? What do you think?